src/main/java/com/maybe/maybe/repository/EmployeeRepository.java

```diff
@@ -4,4 +4,6 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 
 public interface EmployeeRepository extends JpaRepository<Employee, Long> {
+    Employee findEmployeeById(Long id);
```

Comment:
There is a default method findById
which you don't need to describe in repository class
You can use it in services